infra/resources/_modules/cdn/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "storage_account" {
```

Comment:
I was thinking about an approach we took on other repos, and it was always group all NON sensitive properties under one value (only the necessary ones of course), keeping the sensitive ones in their own outputs. For example:

```hcl
output "storage_account" {
  value = {
    id = module.wallet_cdn.id
    name = module.wallet_cdn.name
  }
}

output "storage_account_primary_connection_string" {
  value = module.wallnet_cdn.primary_connection_string
  sensitive = true
}
```

What you've done is ok, I'm just saying